test/core.jl

```diff
@@ -4609,3 +4609,27 @@ end
 
 # issue #19963
 @test_nowarn ccall(:jl_free, Void, (Ptr{Void}, ), C_NULL)
+
+# Wrong string size on 64bits for large string.
+if Sys.WORD_SIZE == 64
+    @noinline function test_large_string20360(slot)
+        try
+            slot[] = Base._string_n(2^32)
```

Comment:
a subset of that would be good as comments so people don't mess up the try-catch by accident